### ISSUE TITLE
The compute_final_scores script erroneously sees some authors as judges

### ISSUE THREAD

**Issue by jmacdotorg** [iftechfoundation/ifcomp#197]
**Deadline: November 15**

If an author has entered more than one game into the current competition, and at least one of those games marked `is_disqualified` in the database, and at least one of those games is *not* marked `is_disqualified` in the database, then `compute_final_scores.pl` will erroneously consider that author to be a judge. This can result in incorrect scores.

(The band-aid for this involves straight-up deleting disqualified game records from the database prior to running this script.)

**Comment by jmacdotorg** [comment]
This needs to be delivered by November 15.

**Comment by jmacdotorg** [comment]
On investigating this further, I found that that I didn't quite have the problem right. The problem is not that authors with mixed DQ and non-DQ games get counted as judges; the problem is that non-authors with more than one DQed game get their vote-counts multiplied by the number of DQed entries they have in the current comp.

The PR I just submitted fixes this by adding a group-by clause that flattens this effect.
